README.md

```diff
@@ -150,7 +150,14 @@ Various class names are attached to the sidebar and container for easier styling
 | Property name | Type | Default | Description |
 | ------------- | ---- | ------- | ----------- |
 | backdropClass | string | | Additional class name on the overlay element. |
+| showBackdrop | boolean | `false` | Controls the backdrop state of the sidebar container. This should be two-way bound. |
+| allowSidebarBackdropControl | boolean | `true` | Determines if the container component respects the sidebar's `showBackdropOnOpen` input. |
```

Comment:
I guess this (long) name makes the most sense. Just make sure `showBackdropOnOpen` is changed to `showBackdrop`.